course/models.py

```diff
@@ -700,25 +696,25 @@ class CourseModule(UrlMixin, models.Model):
     opening times and deadlines for exercises.
     """
     STATUS = Enum([
-        ('READY', 'ready', _("Ready")),
-        ('UNLISTED', 'unlisted', _("Unlisted in table of contents")),
-        ('HIDDEN', 'hidden', _("Hidden")),
-        ('MAINTENANCE', 'maintenance', _("Maintenance")),
+        ('READY', 'ready', _('STATUS_READY')),
+        ('UNLISTED', 'unlisted', _('STATUS_UNLISTED')),
+        ('HIDDEN', 'hidden', _('HIDDEN_capitalized')),
```

Comment:
Should this be STATUS_HIDDEN, to be consistent with the other enum values?

Reply:
I considered that, but then we need several keys that match to the word "Hidden". However, that might be good if one wants to change the phrase or word displayed instead of hidden in the status context but not everywhere else. Which do you think would be better?